cookbook/service_container/event_listener.rst

```diff
@@ -74,30 +77,133 @@ using a special "tag":
         # app/config/services.yml
         services:
             kernel.listener.your_listener_name:
-                class: AppBundle\EventListener\AcmeExceptionListener
+                class: AppBundle\Listener\ExceptionListener
                 tags:
-                    - { name: kernel.event_listener, event: kernel.exception, method: onKernelException }
+                    - { name: kernel.event_listener, event: kernel.exception }
 
     .. code-block:: xml
 
         <!-- app/config/services.xml -->
-        <service id="kernel.listener.your_listener_name" class="AppBundle\EventListener\AcmeExceptionListener">
-            <tag name="kernel.event_listener" event="kernel.exception" method="onKernelException" />
+        <service id="kernel.listener.your_listener_name" class="AppBundle\Listener\ExceptionListener">
+            <tag name="kernel.event_listener" event="kernel.exception" />
         </service>
 
     .. code-block:: php
 
         // app/config/services.php
         $container
-            ->register('kernel.listener.your_listener_name', 'AppBundle\EventListener\AcmeExceptionListener')
-            ->addTag('kernel.event_listener', array('event' => 'kernel.exception', 'method' => 'onKernelException'))
+            ->register('kernel.listener.your_listener_name', 'AppBundle\Listener\ExceptionListener')
+            ->addTag('kernel.event_listener', array('event' => 'kernel.exception'))
         ;
 
 .. note::
 
-    There is an additional tag option ``priority`` that is optional and defaults
-    to 0. The listeners will be executed in the order of their priority (highest to lowest).
-    This is useful when you need to guarantee that one listener is executed before another.
+    There is an optional tag attribute called ``method`` which defines which method
+    to execute when the event is triggered. By default the name of the method is
+    ``on`` + "camel-cased event name". If the event is ``kernel.exception`` the
+    method executed by default is ``onKernelException()``.
+
+    The other optional tag attribute is called  ``priority``, which defaults to
+    ``0`` and it controls the order in which listeners are executed (the highest
+    the priority, the earlier a listener is executed). This is useful when you
+    need to guarantee that one listener is executed before another. The priorities
+    of the internal Symfony events range from ``-255`` to ``255`` but your own
```

Comment:
events don't have a priority. Listeners do. And we have listeners with a priority higher than 255 in the core, so the message is wrong